sysdump/sysdump.go

```diff
@@ -97,34 +97,38 @@ func Collect(c *k8s.Client, o Options) error {
 	// KubeArmor Pod
 	errs.Go(func() error {
 		pods, err := c.K8sClientset.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
-			LabelSelector: "kubearmor-app=kubearmor",
+			LabelSelector: "kubearmor-app",
 		})
+
 		if err != nil {
 			fmt.Printf("kubearmor pod not found. (possible if kubearmor is running in process mode)\n")
```

Comment:
Let's update the print statement to inform the user that it is checking all the KubeArmor pods.